clusterloader2/cmd/clusterloader.go

```diff
@@ -49,11 +48,11 @@ var (
 )
 
 func initClusterFlags() {
-	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "", "Path to the kubeconfig file")
-	pflag.IntVar(&clusterLoaderConfig.ClusterConfig.Nodes, "nodes", 0, "number of nodes")
-	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.Provider, "provider", "", "Cluster provider")
-	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.MasterName, "mastername", "", "Name of the masternode")
-	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.MasterIP, "masterip", "", "Hostname/IP of the masternode")
+	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "KUBECONFIG", "", "Path to the kubeconfig file")
+	flags.IntEnvVar(&clusterLoaderConfig.ClusterConfig.Nodes, "nodes", "NODES", 0, "number of nodes")
```

Comment:
Do we really need this env?

Reply:
If so, isn't that NUM_NODES ?

Reply:
I assumed that there possible should be env variable for each cluster flag.
Changed to "NUM_NODES".